MADE.App.Views.Controls.HeaderedTextBlock.Forms/MADE.App.Views.Controls.HeaderedTextBlock.Forms.csproj

```diff
@@ -25,6 +25,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta-63127-02" PrivateAssets="All" />
     <PackageReference Include="Xamarin.Forms" Version="3.6.0.220655" />
+    <PackageReference Include="XPlat.UI.Controls" Version="1.6.19073.2-alpha" />
```

Comment:
As we are using the pre-release versions of your XPlat packages, are we going to ensure that the official release is available before we get MADE to 1.0?

Reply:
So I've created milestones for 1.6 of XPlat and 1.0 of MADE which should be available on the same day.